code/backend/src/main/java/server/rest/controllers/SkillsController.java

```diff
@@ -0,0 +1,48 @@
+package server.rest.controllers;
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import server.database.DatabaseConnection;
+import server.model.Skill;
+import server.rest.responses.SkillsResponse;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@CrossOrigin(origins = "http://localhost:1234")
```

Comment:
See others.

Reply:
@StephenVanSon  Can you make the changes to the origins after I merging this? I don't think I can get that done today

Reply:
Yup I can